.vscode/c_cpp_properties.json

```diff
@@ -0,0 +1,20 @@
+{
+  "configurations": [
+    {
+      "name": "Linux",
+      "includePath": [
+        "${workspaceFolder}/**",
+        "/usr/include/glib-2.0",
+        "${HOME}/.node-gyp/12.9.1/include/node",
+        "/usr/lib/x86_64-linux-gnu/glib-2.0/include",
+        "/usr/include/gobject-introspection-1.0"
+      ],
+      "defines": [],
+      "compilerPath": "/usr/bin/clang-6.0",
```

Comment:
Is it possible to point to `/usr/bin/clang`? Otherwise this config will not work for people with different clang version :)

Reply:
yes.. that right... i forgot that... then we use `g++` ???

Reply:
It depends, now that I think it might be better not to include this because people migth use different compilers :/

Reply:
Ya, that's right... wait a minute ... I don't think this will have an impact on this project ... this config is only for Intellisense, include directories.
Because for `build` we use` binding.gyp` right?

Reply:
This compiler will only be triggered if we do the `build` action in VSCode

Reply:
Yes, but I mean if other developpers come accross the project and open it in VSCode, they will get this config by default, which is not necessarily good. What about we include it in the readme or on another markdown document as an example?

Reply:
Ya... That's good idea !!!

Reply:
so i close it ??